README.rst

```diff
@@ -1,12 +1,24 @@
 Description
 ===========
 
-Drizzle Redux is a project which aims to breath new life into the libdrizzle
+Drizzle Redux is a project which aims to breathe new life into the libdrizzle
 C connector. It is designed to allow you to connect to and query a
 MySQL database server using a simple API.
 
-Libdrizzle-redux in the sociomantic repository is a fork of the official
-libdrizzle redux at https://launchpad.net/libdrizzle.
+Libdrizzle-redux in the sociomantic tsunami repository is a fork of the official
+**v5.1.4 Felmingham** at https://launchpad.net/libdrizzle.
+
+The primary reason for the fork is that development of the upstream stopped in 2013 
+which meant that crucial fixes and enhancements  were not included in the latest 
+stable release **v5.1.4 Felmingham**.
+
+We hope that continuing development of libdrizzle-redux can benefit current and future 
+users of the library. 
+
+In addition to bug-fixes, the most notable difference from upstream is the ability to 
```

Comment:
This is still correct at the moment but will not always be if we need to add in more binlog non-blocking stuff.